lib/kitchen/command.rb

```diff
@@ -170,17 +170,25 @@ def run_action(action, instances, *args)
         concurrency.times do
           threads << Thread.new do
             while instance = queue.pop
+              puts "running #{instance.name}"
```

Comment:
Oops.. that should be pulled.  I was troubleshooting the difference between actionfailed and instancefailed.  Thanks.